homeassistant/components/doorbird.py

```diff
@@ -113,7 +114,16 @@ def setup(hass, config):
 
         # Subscribe to doorbell or motion events
         if events:
-            doorstation.update_schedule(hass)
+            try:
+                doorstation.update_schedule(hass)
+            except HTTPError:
+                hass.components.persistent_notification.create(
```

Comment:
But should we not `return False`? We say they need to fix and restart. Why should we set up the component after this error? And components that fail set up already get a persistent notification.

Reply:
Technically, the component isn't affected by this failure since it is simply a failure to confiure the remote device, but I have made the change as requested.